style.css

```diff
@@ -10,15 +28,95 @@
 
 .pollsection {
 	border-collapse: collapse;
+    width: 100%;
+	margin-bottom: 60px;
+}
+
+.pollsection tr:nth-child(even) {
+	background-color: #eef;
 }
 .pollsection th,td {
-	border: 1px solid black;
 	padding: 0.5ex 1ex;
 }
-.poll th {
-	text-align: left;
+th:first-child {
+	text-align: left !important;
```

Comment:
Yes, this rule applies only to the first child (the 'Do you agree with this' text), all the rest is centered.